app/models/user.rb

```diff
@@ -42,10 +42,8 @@ def send_invitation(user_id)
   end
 
   def confirm_invites(user)
-    send_invitation.update_attributes(confirmed: true)
-    Friendship.create!(friend_id: send_invitation.user_id,
-                  user_id: send_invitation.friend_id,
-                  confirmed: true)
```

Comment:
What you had here was 'better' in a sense. What was missing is the fact that you tried to run update attributes on a method. Instead, you should try to find the inverse friendship corresponding to the current user and the user in question and update that. Then create an inverse friendship like you attempted to in 45-48

Reply:
```ruby
def confirm_friend
  self.update_attributes(confirmed: true)
  Friendship.create!(friend_id: self.user_id,
                  user_id: self.friend_id,
                  confirmed: true)
end
```

Reply:
This should be done on the friendship